src/ocean/text/Arguments.d

```diff
@@ -1278,6 +1278,7 @@ public class Arguments
 
         Returns:
             integer value of the parameter assigned to the argument
+            0 if value is missing or not a valid integer
```

Comment:
Obviously we need to fix the regression for now, but is this a desirable behaviour?  It looks like this makes it impossible to tell the difference between `--arg 0` and `--arg somethinginvalid`.

Maybe we should look at updating this behaviour in the next major release?

Reply:
Indeed. It is pretty odd behaviour. Currently, you need to do this:
```D
if ( arg.assigned.length > 0 )
{
  arg.getInt(...);
}
else
  // undefined
```

Reply:
Looking at the offending turtle's usage [1]:

```
 auto progress = args.getInt!(long)("progress");
```
I think this should be written as:

```
auto progress = args.exist("progress")? args.getInt!(long)("progress") : 0;
```

if it doesn't care about it, or it should have specified the default value.

[1] https://github.com/sociomantic-tsunami/turtle/blob/v9.x.x/src/turtle/runner/Runner.d#L692